templates/repo/blame.tmpl

```diff
@@ -16,11 +16,13 @@
 				{{end}}
 				<a class="ui tiny button" href="{{.RepoLink}}/src/{{EscapePound .BranchNameSubURL}}/{{EscapePound .TreePath}}">{{.i18n.Tr "repo.normal_view"}}</a>
 				<a class="ui tiny button" href="{{.RepoLink}}/commits/{{EscapePound .BranchNameSubURL}}/{{EscapePound .TreePath}}">{{.i18n.Tr "repo.file_history"}}</a>
+				<a class="ui tiny button unescape-button">{{.i18n.Tr "repo.unescape_control_characters"}}</a>
+				<a class="ui tiny button escape-button" style="display: none;">{{.i18n.Tr "repo.escape_control_characters"}}</a>
 			</div>
 		</div>
 	</h4>
 	<div class="ui attached table unstackable segment">
-		<div class="file-view code-view">
+		<div class="file-view code-view escaped">
```

Comment:
As below.

The word `escaped` is quite short and may be misleading. Maybe `unicode-escaped` is more clear.